common/build/eslint-plugin-fluid/src/test/mockFiles/no-unchecked-record-access/fileWithOnlyRecordAccess.ts

```diff
@@ -0,0 +1,13 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const record = { a: 1, b: 2 };
+const record1 = record.a; // This should not report an error
+const recordB = record.b; // This should not report an error
```

Comment:
nit: if no reason to name them differently, please name things consistently. `record1` vs `recordB`